packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -727,6 +727,12 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
   }
 
   void _handleDragCancel() {
+    if (_gestureDetectorKey.currentContext == null) {
```

Comment:
Is this change related?

Reply:
Yeah, without this, the `test/widgets/scrollable_dispose_test.dart` fails because as part of cancelling the hold or drag below the scrollable wants to go ballistic, for which it needs to know the device pixel ratio for one of its calculations. Doing that now requires an inherited widget lookup, which fails if the we are in the process of being disposed. That's fine though, since we can just skip going ballistic if we get disposed anyways.

Reply:
Oh I see, thank you!